blockindex/bloomfilterindexer.go

```diff
@@ -282,6 +274,7 @@ func (bfx *bloomfilterIndexer) calculateBlockBloomFilter(ctx context.Context, re
 	return bloom
 }
 
+// TODO: improve performance
```

Comment:
`// TODO (liuhaai): improve performance`

Reply:
IMO, this is a good independent task for beginners. I will file it as a good-frist-issue issue.